wrapanapi/systems/msazure.py

```diff
@@ -793,25 +811,39 @@ def remove_pips_by_search(self, pip_template, resource_group=None):
             operation.wait()
             self.logger.info('"%s" pip removed', pip)
             results.append((pip, operation.status()))
-        if not results:
+        if pip_template and not results:
```

Comment:
Same as above, this message should be written regardless of `pip_template` being empty.

Reply:
Removed checking of pip_template availability and throwing a general mssg if no results though its pip_template or all.